sympy/matrices/common.py

```diff
@@ -2019,7 +2019,7 @@ class MatrixOperations(MatrixRequired):
     operations.  Should not be instantiated directly."""
 
     def _eval_adjoint(self):
-        return self.transpose().conjugate()
+        return self.transpose().applyfunc(lambda x: x.adjoint())
```

Comment:
These classes aren't used any more. The change should be in `matrixbase.py`.

Reply:
I did also make the change in `matrixbase.py`. I implemented this one because I assumed that it makes sense to also fix the bug in the deprecated class.

I now reverted the change to `MatrixOperations`.